transport/http/outbound.go

```diff
@@ -524,6 +523,9 @@ func getYARPCErrorFromResponse(tres *transport.Response, response *http.Response
 	).WithDetails(details)
 
 	if bothResponseError {
+		if response.StatusCode >= 400 {
+			UpdateSpanWithErrAndCode(span, yarpcErr, yarpcerrors.FromError(yarpcErr).Code())
```

Comment:
We have "code" var here, why do we need to get it via creating new error?

Reply:
Why do we even need it, if UpdateSpanWithErrAndCode already was called unconditionally at transport/http/outbound.go::317?